<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceSettingsWidget</class>
 <widget class="QWidget" name="InterfaceSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>617</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="behaviourGroup">
     <property name="title">
      <string>Behaviour</string>
     </property>
     <layout class="QGridLayout" name="formLayout_4">
      <item row="1" column="0">
       <widget class="QCheckBox" name="inhibitScreensaver">
        <property name="text">
         <string>Inhibit Screensaver</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="confirmShutdown">
        <property name="text">
         <string>Confirm Shutdown</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="discordPresence">
        <property name="text">
         <string>Enable Discord Presence</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="mouseLock">
        <property name="text">
         <string>Enable Mouse Lock</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="pauseOnStart">
        <property name="text">
         <string>Pause On Start</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="pauseOnControllerDisconnection">
        <property name="text">
         <string>Pause On Controller Disconnection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="pauseOnFocusLoss">
        <property name="text">
         <string>Pause On Focus Loss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gameDisplayGroup">
     <property name="title">
      <string>Game Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="startFullscreen">
        <property name="text">
         <string>Start Fullscreen</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="doubleClickTogglesFullscreen">
        <property name="text">
         <string>Double-Click Toggles Fullscreen</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="renderToSeparateWindow">
        <property name="text">
         <string>Render To Separate Window</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="hideMainWindow">
        <property name="text">
         <string>Hide Main Window When Running</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="disableWindowResizing">
        <property name="text">
         <string>Disable Window Resizing</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="hideMouseCursor">
        <property name="text">
         <string>Hide Cursor In Fullscreen</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="startFullscreenUI">
        <property name="text">
         <string>Start Big Picture UI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="appearancesGroup">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QComboBox" name="theme"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="languageLabel">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="language"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="themeLabel">
        <property name="text">
         <string>Theme:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="backgroundLabel">
        <property name="text">
         <string>Game List Background:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="backgroundPathLayout">
        <item>
         <widget class="QPushButton" name="backgroundBrowse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="backgroundReset">
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="icon">
           <iconset theme="trash-fill"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="backgroundOpacityLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Opacity:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="backgroundOpacity">
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string notr="true">%</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="backgroundFill">
          <property name="text">
           <string>Fill Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="automaticUpdaterGroup">
     <property name="title">
      <string>Automatic Updater</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="updaterChannelLabel">
        <property name="text">
         <string>Update Channel:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="autoUpdateCurrentVersion">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="autoUpdateTag"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="currentVersionLabel">
        <property name="text">
         <string>Current Version:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="automaticUpdaterCheckGroup" stretch="1,0">
        <item>
         <widget class="QCheckBox" name="autoUpdateEnabled">
          <property name="text">
           <string>Enable Automatic Update Check</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="checkForUpdates">
          <property name="text">
           <string>Check for Updates...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
